src/fides/api/alembic/migrations/versions/14acee6f5459_translation_data.py

```diff
@@ -590,19 +594,10 @@ def create_new_experience_config(experience_config):
                 "history_id": generate_record_id("pri"),
                 "version": 1,
                 "is_default": True,
+                "language": "en",
             },
         )
 
-        # backfill language column in all experience config history records
-        update_experience_config_history_language = text(
-            """
-            UPDATE privacyexperienceconfighistory
-            SET language = 'en'
-        """
-        )
-
-        bind.execute(update_experience_config_history_language)
-
```

Comment:
No longer backfilling language across all historical records.  They will only be set on new ones created here and new records going forward.